paysage/metrics.py

```diff
@@ -394,3 +394,139 @@ def value(self) -> float:
             return self.calc.var
         else:
             return None
+
+
+class CrossEntropy(object):
+    """
+    Compute the root-mean-squared error between observations and their
```

Comment:
fix the docstrings in this file, they are all duplicates of the `ReconstructionError` class.